backend/models/submission.js

```diff
@@ -0,0 +1,81 @@
+const mongoose = require("mongoose");
+
+const matrix = new mongoose.Schema({
+  x: {
+    type: Number
+  },
+  y: {
+    type: Number
+  }
+});
+
+const option = new mongoose.Schema({
+  min: {
+    type: Number
+  },
+  max: {
+    type: Number
```

Comment:
Could someone remind me what this is for 😅

Reply:
Honestly, now that I look back at it I'm a little confused as well. I think it's related to the "id" for each option in an MC/checkbox question but hoping @GodGreg can confirm.

Reply:
Ok! I'm sure there's a good reason, would be great if the naming/comments could clarify

Reply:
This is for formSchema and I don't think it should exist here

This relates to the number of choices that need to selected (multiple choice and check box). 

We shouldn't need options at all